eng/Packages.Data.props

```diff
@@ -48,7 +48,7 @@
     <PackageReference Update="System.Security.Cryptography.Cng" Version="4.3.0" />
     <PackageReference Update="System.Security.Cryptography.Primitives" Version="4.3.0" />
     <PackageReference Update="System.Text.Encodings.Web" Version="4.5.1" />
-    <PackageReference Update="System.Text.Json" Version="4.7.2" />
+    <PackageReference Update="System.Text.Json" Version="7.0.0" />
```

Comment:
This impacts all SDKs. Curious, why this change is needed. Same for above version of System.Memory.Data